src/app/modules/gb-navbar-module/gb-navbar.module.ts

```diff
@@ -12,15 +12,16 @@ import {GbNavbarComponent} from './gb-navbar.component';
 import {RouterModule} from '@angular/router';
 import {TabMenuModule} from 'primeng/components/tabmenu/tabmenu';
 import {FormsModule} from '@angular/forms';
-import {MessagesModule} from 'primeng/primeng';
+import {MessagesModule, TooltipModule} from 'primeng/primeng';
```

Comment:
are we using tooltip in navbar?

Reply:
Yes, to inform users that IE is not supported.